website/app/components/doc/related-components/related-components.hbs

```diff
@@ -0,0 +1,5 @@
+{{! hr is temporary }}
+<hr />
+
+<h2 class="doc-text-h2">Related</h2>
+<Doc::Cards::Deck @cols="2" @layout="horizontal" @cards={{this.relatedComponents}} />
```

Comment:
I just pasted in the existing code from show.hbs to start. I can't figure out how to get it to recognize the new component though.